app/views/aircrafts/index.html.erb

```diff
@@ -0,0 +1,40 @@
+<div class="container mt-3">
+  <div class="row">
+    <div class="col-12">
+      <h1>All our Aircrafts</h1>
```

Comment:
title can just be "Fly for a day or two?" or "All Aircraft" "our aircraft" doesn't make sense